scripts/log-session.py

```
@@ -0,0 +1,334 @@
+
+        # Write markdown
+        log_file = get_today_log()
+        markdown = format_markdown(entry)
+        with open(log_file, 'a') as f:
+            f.write(markdown)
+
+        # Emit event for FS5 (GAP-5)
+        emit_event(entry)
+
+        print(f'\n[OK] Entry logged to {log_file}')
+        print(f'[OK] Event emitted to memory/events.jsonl')
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)